src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -0,0 +1,2201 @@
+	private JButton printButton;
+	private JButton deleteButton;
+	private JButton saveButton;
+	private JButton resetButton;
+	private JButton lotButton;
+	private JButton validateButton;
+	private JButton confirmButton;
+	private JScrollPane scrollPaneInventory;
+	private JTable jTableInventoryRow;
+	private DefaultTableModel model;
+	private List<MedicalInventoryRow> inventoryRowList = new ArrayList<>();
+	private List<MedicalInventoryRow> inventoryRowSearchList = new ArrayList<>();
+	private List<MedicalInventoryRow> inventoryRowsToDelete = new ArrayList<>();
+	private List<MedicalInventoryRow> inventoryRowListAdded = new ArrayList<>();
+	private List<Lot> lotsSaved = new ArrayList<>();
+	private List<Lot> lotsDeleted = new ArrayList<>();
+	private String[] columsNames = { MessageBundle.getMessage("angal.inventory.id.col").toUpperCase(),
+			MessageBundle.getMessage("angal.common.code.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.medical.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.newlot.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.lotcode.col").toUpperCase(),
+			MessageBundle.getMessage("angal.medicalstock.duedate.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.theoreticalqty.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.realqty.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.unitcost.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.totalcost.col").toUpperCase()
+	};
+	private int[] columwidth = { 50, 50, 200, 100, 100, 100, 100, 80, 80, 80 };
+	private boolean[] columnEditable = { false, false, false, false, false, false, false, true, false, false };
+	private boolean[] columnEditableView = { false, false, false, false, false, false, false, false, false, false };
+	private boolean[] columnVisible = { false, true, true, true, !GeneralData.AUTOMATICLOT_IN, true, true, true, GeneralData.LOTWITHCOST,
+			GeneralData.LOTWITHCOST };
+	private boolean[] columnCentered = { false, false, false, true, true, true, true, true, true, true };
+	private boolean[] columnDecimalNumber = { false, false, false, false, false, false, false, false, true, true };
+	private Class< ? >[] columnsClasses = { String.class, Integer.class, String.class, String.class, String.class, LocalDate.class, Integer.class,
+			Integer.class, BigDecimal.class, BigDecimal.class };
+	private MedicalInventory inventory;
+	private JLabel addMedicalLabel;
+	private JButton selectButton;
+	private JLabel dateInventoryLabel;
+	private JTextField medicalCodeTextField;
+	private String code;
+	private String mode;
+	private JLabel statusLabel;
+	private JLabel referenceLabel;
+	private JLabel chargeTypeLabel;
+	private JLabel dischargeTypeLabel;
+	private JLabel supplierLabel;
+	private JLabel addMedicalsByLabel;
+	private JLabel destinationLabel;
+	private JTextField referenceTextField;
+	private JTextField jTextFieldEditor;
```

Comment:
Move the variable  into  the method `getJTableInventoryRow()` as that is the only place it is used